source/llassetgen/source/DistanceTransform.cpp

```diff
@@ -219,6 +219,41 @@ namespace llassetgen {
                 if (inputAt({x, y})) outputAt({x, y}) *= -1;
     }
 
+
+
+    template<bool fill>
+    void ParabolaEnvelope::edgeDetection(DimensionType offset, DimensionType pitch, DimensionType length) {
+        InputType prev = 0;
+        DimensionType begin = 0;
+        for(DimensionType j = 0; j < length; ++j) {
+            InputType next = inputAt(offset+j*pitch);
+            if(next == prev)
```

Comment:
When is this possible?

Reply:
The loop ignores a pixel, when it has the same value (boolean) as the previous one. (Hence the name edge detection)